aws/chef-bootstrap.sh

```diff
@@ -112,9 +112,17 @@ ln -fs ${SHUTDOWN_SH} /etc/rc0.d/K04chef_shutdown
 ln -fs ${SHUTDOWN_SH} /etc/rc6.d/K04chef_shutdown
 
 if [ "${LOCAL_MODE}" = "1" ]; then
+
+  # Keep this in sync with CHEF_KEY constant in cloud_formation.rb
+  if [ "${ENVIRONMENT}" = "adhoc" ]; then
+    CHEF_KEY=adhoc/chef
+  else
+    CHEF_KEY=chef
+  fi
```

Comment:
I figured it was simpler to just inline this logic into the shell script rather than start processing this script via a ruby-template just for this one conditional variable.